cli/src/main/java/org/jboss/as/cli/impl/CliLauncher.java

```diff
@@ -21,11 +21,8 @@
  */
 package org.jboss.as.cli.impl;
 
-import java.io.BufferedReader;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.FileReader;
+import java.io.*;
```

Comment:
Please restore the non-wildcard imports as that's the WildFly codestyle.

Reply:
done